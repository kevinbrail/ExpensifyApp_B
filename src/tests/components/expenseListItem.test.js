import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../components/expenseListItem';
import expenses from '../fixtures/expenses';


test('Should render an individual expense', () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[0]} />)
    expect(wrapper).toMatchSnapshot();
});