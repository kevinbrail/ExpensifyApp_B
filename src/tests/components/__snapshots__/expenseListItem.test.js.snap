// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render an individual expense 1`] = `
<div>
  <h3>
    Expense
  </h3>
  <h3>
     
    DESC1
     
    <Link
      replace={false}
      to="/edit/1"
    >
      (EDIT)
    </Link>
  </h3>
  <p>
    $
    2001
     
    0
       
    NOTE
     
    1
  </p>
</div>
`;
